content/news/releases/prestashop-8-2-0-available.md

```diff
@@ -0,0 +1,91 @@
+
+**Fixed installation process on certain environment configurations**
+
+Due to different configurations of some environments, the installation process was failing on the database creation step. This issue [has been fixed](https://github.com/PrestaShop/PrestaShop/pull/36840) in this version.
+
+**Fixed incompatibility issue with PDO returning integer values on PHP 8.1**
+
+In PHP 8.1, PDO started returning integer values for some fields that were previously returned as strings. This change caused compatibility issues with PrestaShop, and it has been [fixed](https://github.com/PrestaShop/PrestaShop/pull/36875) in version 8.2.
+
+**Fixed issue with creating duplicate customers in a specific scenario**
+
+There was an issue where customers could be created multiple times when the same email address was used. This issue has been [fixed](https://github.com/PrestaShop/PrestaShop/pull/36454) and you can read more about it in the pull request.
+
+**Solved some problems with ordering customized products**
+
+Thanks to [@Codencode](https://github.com/Codencode), some issues with ordering customized products have been fixed. They [fixed](https://github.com/PrestaShop/PrestaShop/pull/36834) the issue with the "Total products" column on the order confirmation, and with [deleting images for customized products](https://github.com/PrestaShop/PrestaShop/pull/36905).
+
+## Changed release schedule
+
+We understand that the release of PrestaShop 8.2 may come as a surprise to some, as it deviates from our previously announced release schedule. Initially, our plan was to follow a versioning cycle of 8.0, 8.1, followed by 9.0, 9.1, and beyond. However, in order to fully comply with semantic versioning principles, we have made the decision to introduce this new minor release.
+
+We have also updated the [release schedule page](https://www.prestashop-project.org/project-organization/release-cycle/) on the PrestaShop Project website to reflect those adjustments. While our goal is typically to release one minor version per major release, this update demonstrates our flexibility in issuing additional minor versions when necessary.
+
+Meanwhile, work on PrestaShop 9 is progressing steadily, and we are preparing the first beta release. More details on this upcoming version will be shared in the coming weeks.
+
+## Download
+
+PrestaShop 8.2 is available on GitHub:
+
+{{< cta "https://github.com/PrestaShop/PrestaShop/releases/tag/8.2.0" >}}Download PrestaShop 8.2 now!{{< /cta >}}
+
+## Upgrade
+
+The [1-Click Upgrade module](https://github.com/PrestaShop/autoupgrade) for this version is available. You can download the latest release, enabling the upgrade to 8.2 [directly from GitHub](https://github.com/PrestaShop/autoupgrade/releases). Upgrade for PrestaShop 8.2 is available under "Minor upgrades" channel.
+
+## Acknowledgments
+
+A big thank you to people who contributed to Pull Requests included in this version:
+
+TBD
```

Comment:
Just commenting here for the list of contributors.

Reply:
done 👍